docs/development.md

```diff
@@ -5,33 +5,70 @@ help you get started.
 
 ## Prerequisites
 
-### apiserver-builder
+The federation v2 deployment requires kubernetes version >= 1.11. The following
+is a detailed list of binaries required.
+
+### Binaries
+
+The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
+`kube-apiserver` >= v1.11 being installed in the path. The `kubebuilder`
+release packages all of these dependencies together, but the latest version of
+`kubebuilder` (v0.1.12 as of this writing) only includes binaries for
+kubernetes 1.10. Once `kubebuilder` bundles binaries >= 1.11, it will
+only require downloading the `kubebuilder` release.
+
+These binaries can be installed via the `download-binaries.sh` script, which
+downloads them to `./bin`:
+
+```bash
+./scripts/download-binaries.sh
+export PATH=$(pwd)/bin:${PATH}
+```
+
+Or you can install them manually yourself using the guidelines provided below.
+
+#### kubebuilder
 
 This repo depends on
-[apiserver-builder](https://github.com/kubernetes-incubator/apiserver-builder)
-to generate code and build binaries. Download a [recent
-release](https://github.com/kubernetes-incubator/apiserver-builder/releases)
+[kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+to generate code and build binaries. Download the [v0.1.12
+release](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v0.1.12)
 and install it in your `PATH`.
 
+#### kubectl
+
+Download a version of
+[`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) >= v1.11 and
+install it in your `PATH`.
+
```

Comment:
How about add a link of `kubectl` v1.11 like line 51?

Reply:
I've added it because we have a special dependency on 1.11. Otherwise I don't normally like doing that because it quickly grows stale.